locales/en/app.ftl

```diff
@@ -827,3 +827,9 @@ vpn-banner-cta-close = Close
 relay-video-headline = How do you keep your email address a secret?
 # VPN video headline for placement on Monitor user's dashboard
 vpn-video-headline = Do you worry about safety on public Wi-Fi?
+
+# DATA REMOVAL SPECIFIC
+
+remove-header-exposures = Data Removal
+remove-footer-data-removal-faq = Data Removal FAQ
+remove-faq-title = Data Removal Pilot FAQ
```

Comment:
Please keep an empty line at the end of the file.

Reply:
👍🏽